submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/FileListValidationTest.kt

```diff
@@ -72,13 +82,94 @@ class FileListValidationTest(
 
     @Test
     fun `valid file list`() {
-        val fileListFile = tempFolder.createFile("Plate1.tif")
-        val fileList = tempFolder.createFile("FileList.json", getFileListContent().toString())
+        val previousVersion = tsv {
+            line("Submission", "S-FLV123")
+            line()
 
-        webClient.uploadFiles(listOf(fileListFile, fileList))
-        webClient.validateFileList(fileList.name)
+            line("Study")
+            line()
 
-        webClient.deleteFile(fileListFile.name)
+            line("File", "Plate2.tif")
+            line()
+        }.toString()
+
+        val file1 = tempFolder.createFile("Plate1.tif", "content-1")
```

Comment:
personally I found easier (less cognitive load) file1.txt, file2.txt and like that.